### Notebook Diffs vs origin/main
nbdiff /dev/null newnb.ipynb
--- /dev/null  2025-07-30 12:12:51.210027
+++ newnb.ipynb  2025-07-30 12:12:38.337502
## added /cells:
+  code cell:
+    source:
+      new nb
+  code cell:




